jax/_src/pjit.py

```diff
@@ -1757,6 +1757,20 @@ def _resolve_wsc_args(axis_resources, shardings):
 # period is finished. The deprecation period expires 3 months from Feb 13, 2023.
 def with_sharding_constraint(x, shardings=UNSPECIFIED,
                              axis_resources=UNSPECIFIED):
+  """Mechanism to constrain the sharding of an Array inside a jitted computation
+
+  This is a strict constraint for the GSPMD partitioner and not a hint. For examples
+  of how to use this function, see `Distributed arrays and automatic parallelization`_.
+
+  Args:
+    x: PyTree of jax.Arrays which will have their shardings constrainted
+    shardings: PyTree of jax.sharding.Sharding instances specifying the sharding
+    axis_resources: (deprecated) alternate means of specifying shardings.
```

Comment:
You can say; `axis_resources: (deprecated) use shardings argument`?

Reply:
Done